app/kubernetes/views/applications/applicationsController.js

```diff
@@ -109,6 +152,7 @@ class KubernetesApplicationsController {
 
   async onInit() {
     this.state = {
+      endpointId: this.$transition$.params().endpointId,
```

Comment:
I believe this is related to Chaim latest PR regarding endpoint/URL am I right? This is something we might rollback in the next few days but i'll keep this in mind.

Reply:
It is, I can use EndpointProvider to retrieve the endpoint ID tho as would have been the way to do it without Chaim latest PR on endpointID in URL